lib/denylist/http.go

```diff
@@ -102,12 +102,17 @@ func createDenylistEntry(response http.ResponseWriter, request *http.Request, de
 	denylist.Store(id, true)
 	log.Log.Infow("Created denylist entry", "id", id)
 	metricFilterEnabled.WithLabelValues(id).Set(1)
+	err := syncer.StoreDenylistEntry(denylist, id)
+	if err != nil {
+		http.Error(response, "failed to persist removal of denylist entry", http.StatusInternalServerError)
```

Comment:
This is going to cause us to get sentry alerts; the suggestion from the CT documentation for sentry is to convert this to a metric, set an alert, and log it at sub-error level (which applies generally to things logged as errors)

Reply:
Why is this going to cause sentry errors? This isn't logging an error, it's sending an error back to the calling code as an http 500.

Reply:
Because I read it wrong!